### ISSUE TITLE
[Feature Request]: Tag for gtfs-contact in hubspot for every agency

### ISSUE THREAD

**Issue by vevetron** [cal-itp/customer-success#575]
### Feature Request

In the future we might automate sending emails that your gtfs is expiring.  To do that we need to know who to send it to in an automated fashion.  Thus it'd be valuable if there was a tag on people at agencies that defined them as a good contact person.

Perhaps CS can develop that tag and over time as they work with agencies, can start tagging contacts accordingly.

### Is this request related to a problem?

There isn't a problem yet.  But maybe we can move towards reducing the time CS spends on these minor issues that could just be an automated email rather than needing a human touch.

### Use Case

e.g. 
- Our automated scripts determine LA Metro trains feed is expiring in 31, 14 days
- there is not an existing Airtable ticket
- We create an airtable ticket
- It hits up hubspot for the gtfs contact at LA Metro
- It sends a 31 day email, 14 day email, 7 day email

### Additional Info

None

**Comment by laneymangan** [comment]
@vevetron Hubspot has "association labels" that should be able to work for this use case. The CS team can discuss this further and get back to you!
![Image](https://github.com/user-attachments/assets/1dbbfda0-1120-4ec2-9e9b-72a53f1db167)

**Comment by laneymangan** [comment]
All contacts on the Reports Site distribution list now have the `GTFS Contact` in Hubspot

@AnthonyRollins I believe because the association label was set as a `single label`, the associated companies now also have GTFS contact label since it's a 1-1 association. I am unable to edit the association label, I believe because the label is now in use. I may have to un-label these contacts, edit the association to be a `pair of labels`, and then relabel them. I will have to address this in the new year, but for now at least the contacts have the label

cc @o-ram @vevetron

**Comment by vevetron** [comment]
I'm trying to figure out how to pull this using python and the api.

**Comment by AnthonyRollins** [comment]
@laneymangan I don't think we need to use the association labels here as we already have a `Job Role` property that has a label for `GTFS`

@vevetron  We need to mark all contacts that are GTFS Contacts as `Job Role = GTFS`, and that might be easier to pull those contacts via the Hubspot API.

**Comment by o-ram** [comment]
@AnthonyRollins @laneymangan I read over this and wanted to share that we do have a `Job Role` property but it does **not** have a label for `GTFS`. The closest label we have is a general `Data` option.

Either way, it seems like we would have to do some additional work to ensure Vivek can pull an accurate list, however he does so:
**Option 1**: Stick with Laney's association labels and make the needed changes to fix the issue she identified
**Option 2**: Add GTFS as an option for the job role property and then update that property for the appropriate contacts + make sure that property appears on contact cards.

I can honestly say I have no idea if there is a benefit to doing this as a formal association versus just making use of the existing custom property, so I don't really have an opinion either way, but I think this is where we stand.

**Comment by evansiroky** [comment]
I'm cool with the existing roles in Hubspot and made this map to the kind of contacts we want to collect at transit agencies with the help of Caltrans District Staff:

- CEO / General Manager – Hubspot role of “Executive”
- CFO – Hubspot role of “Finance”
- A contact related to receiving federal grant money – Hubspot role of “Grants”
- A transit planning contact – Hubspot role of “Planner”
- An IT contact able to answer questions about transit data – Hubspot role of “Data”

**Comment by laneymangan** [comment]
Since we seem to be in agreement to use the `Job Role` property and keep the options as is, I've assigned the `Data` job role property to all contacts on our reports list. A list of all contacts with the `Data` job role property can be found [here](https://app.hubspot.com/contacts/5519226/objectLists/503/filters). This is an active list that will be updated as we continue to assign this role. 

If anyone thinks of any other lists or info from Caltrans District staff that we can use to assign more contacts--let me know! (@evansiroky @vevetron)

In the meantime, Account Managers can make a point to assign/update this role as they interact with agencies, outreach for TDQ tickets etc. I can discuss this more with the AMs but for now @o-ram let me know if you have thoughts on this ask.

**Comment by evansiroky** [comment]
This is great, thanks!
